### DISCUSSION TITLE
Add toram boot option

### DISCUSSION THREAD

**Discussion by theobouwman** [linux-surface/linux-surface#823] [Q&A]
Hi All,

I just created a bootable ubuntu usb drive. But how can I add the toram boot option so I can remove it during installation?

Thanks.

**Comment by qzed** [comment]
You should be able to temporarily edit the kernel command line via the boot loader. For example, on GRUB you can do this by pressing "e".

**Reply by theobouwman** [reply]
since my surface pro 5 only has one usb device (where the boot stick is in) I can't use the keyboard right?

**Reply by theobouwman** [reply]
Because the surface keyboard can't be used right?

**Reply by qzed** [reply]
Ah I see. If you have a type cover then that should work, but if you don't then you'll have to edit the default parameters on your boot drive somehow. Unfortunately I don't have any experience with that.

**Comment by NP-chaonay** [comment]
If you dont have type cover then Idk if this work
1) plug usb bootable
2) wait for surface to boot grub (or other bootloader) EFI
3) wait for 1-2s delay (up to you but normally after every element of CUI/GUI is shown, it is safe to remove maybe)
4) plug out usb and put keyboard instead
5) type the cmdline
6) plug out kbd and plug in usb then wait for 1-3s delay
7) boot up
